src/SprykerEco/Yves/Computop/Mapper/Init/PostPlace/PayPalExpressMapper.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Computop\Mapper\Init\PostPlace;
+
+use Generated\Shared\Transfer\ComputopPayPalExpressPaymentTransfer;
+use Generated\Shared\Transfer\CustomerTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Yves\Router\Router\Router;
+use SprykerEco\Shared\Computop\ComputopConfig as ComputopSharedConfig;
+use SprykerEco\Shared\Computop\Config\ComputopApiConfig;
+use SprykerEco\Yves\Computop\Mapper\Init\AbstractMapper;
+use SprykerEco\Yves\Computop\Plugin\Router\ComputopRouteProviderPlugin;
+
+class PayPalExpressMapper extends AbstractMapper
+{
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\ComputopPayPalExpressPaymentTransfer
+     */
+    public function createComputopPaymentTransfer(QuoteTransfer $quoteTransfer)
```

Comment:
return type

Reply:
cannot be declared because of 7.3 + stan + abstract method